images/base/entrypoint

```diff
@@ -18,6 +18,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+configure_proxy() {
+  # Don't use proxy for the local network
+  LOCAL_NETWORK=$(ip route list dev eth0 proto kernel | awk '{ print $1 }')
```

Comment:
we don't need this

Reply:
don´t we? I think we have to add the docker network to NO_PROXY

Reply:
Kind computes this and adds it